docs/data/toolpad/studio/getting-started/support.md

```diff
@@ -1,50 +1,42 @@
+2. Then, if no duplicates exist, [open an issue](https://github.com/mui/toolpad/issues/new/choose) in the Toolpad repository.
 
-If you think you've found a bug in the codebase, or you have an idea for a new feature, please [search the issues on GitHub](https://github.com/mui/toolpad/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aclosed) before opening a new one, to ensure you're not creating a duplicate.
+### New issue guidelines
 
-- [Open an issue on Toolpad Studio](https://github.com/mui/toolpad/issues/new/choose) for issues or feature requests.
-- [Start a GitHub discussion](https://github.com/mui/toolpad/discussions) for a broad topic. Here you can engage with the team and the discussion would likely result in one or more GitHub issues.
-
-#### New issue guidelines
-
-- Please follow one of the issue templates provided on GitHub.
+- Please follow one the issue templates provided on GitHub.
 - Please use a succinct description that helps others find similar issues.
   - ❌ _"It doesn't work"_
   - ✅ _"Add support for {{new feature}}"_
 - Please don't group multiple topics in one issue.
 - Please don't comment "+1" on an issue. It spams the maintainers and doesn't help move the issue forward. Use GitHub reactions instead (👍).
 
-### Discord
-
-Toolpad Studio is also present on Discord, you can connect with other Toolpad Studio users and get help from the community. To join, simply click on this [Discord server](https://mui.com/r/discord/) and follow the instructions. Looking forward to seeing you there!
-
-:::warning
-Please note that there is a single Discord server for all MUI products and you should post in the Toolpad Studio channel.
-:::
+### Bug reproductions
 
-### Stack Overflow
+We require bug reports to be accompanied by a **minimal reproduction**.
+It significantly increases the odds of fixing the problem.
 
-Stack Overflow contains a wealth of information about MUI products. Toolpad Studio is a new product and you can start by [posting a question](https://stackoverflow.com/questions/tagged/toolpad) to get help from community experts as well as maintainers.
+## Community
 
 ### Social media
 
 Toolpad Studio has a growing presence on [Twitter/X](https://x.com/Toolpad_).
 You can use it to share what you're working on and connect with other developers.
 
-Please keep in mind that we don't actively monitor direct messages on the company's social media accounts, so this is _not_ a good way to get in touch with us directly.
+### Discord
 
-### Direct email
+Toolpad Studio is also present on Discord, you can connect with other Toolpad Studio users and get help from the community. To join, simply click on this [Discord server](https://mui.com/r/discord/) and follow the instructions. Looking forward to seeing you there!
 
-If email is your thing, the team can be directly reached at toolpad@mui.com.
+:::warning
+Please note that there is a single Discord server for all MUI products and you should post in the "toolpad‑studio" channel.
```

Comment:
I had to get creative for this one https://github.com/w3c/csswg-drafts/issues/3434